packages/reports/addon/templates/components/filter-values/range-input.hbs

```diff
@@ -1,24 +1,24 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#if @isCollapsed}}
   {{#if @filter.validations.attrs.values.isInvalid}}
     <FilterValues::InvalidValue/>
   {{else}}
     {{concat @filter.values.firstObject " and " @filter.values.lastObject}}
   {{/if}}
 {{else}}
-  <Input
-    @value={{readonly @filter.values.firstObject}}
-    @placeholder={{this.lowPlaceholder}}
-    class="filter-values--range-input__input {{if @filter.validations.attrs.values.isInvalid "filter-values--range-input__input--error"}}"
-    @key-up={{action "setLowValue"}}
-  />
+  <div class="filter-values--range-input" ...attributes>
+    <input
+      value={{@filter.values.firstObject}}
+      placeholder={{this.lowPlaceholder}}
+      class="filter-values--range-input__input {{if @filter.validations.attrs.values.isInvalid "filter-values--range-input__input--error"}}"
+      {{on "input" (action this.setLowValue value="target.value")}}>
 
-  <span class="filter-values--range-input__separator">and</span>
+    <span class="filter-values--range-input__separator">and</span>
 
-  <Input
-    @value={{readonly @filter.values.lastObject}}
-    @placeholder={{this.highPlaceholder}}
-    class="filter-values--range-input__input {{if @filter.validations.attrs.values.isInvalid "filter-values--range-input__input--error"}}"
-    @key-up={{action "setHighValue"}}
-  />
-{{/if}}
\ No newline at end of file
+    <input
+      value={{@filter.values.lastObject}}
+      placeholder={{this.highPlaceholder}}
+      class="filter-values--range-input__input {{if @filter.validations.attrs.values.isInvalid "filter-values--range-input__input--error"}}"
+      {{on "input" (action this.setHighValue value="target.value")}}>
```

Comment:
I've never seen this `value="..."` syntax with the action helper before. Is there any way to get the same functionality using the new `fn` helper instead?

Reply:
You can't accomplish the same thing just via the `fn` helper, but we could use `pipe` to read in the event, map it to `target.value`, and then pass it to `setHighValue`
```suggestion
      {{on "input" (pipe (get _ "target.value") this.setHighValue)}}>
```

Reply:
saw it in https://github.com/emberjs/ember.js/blob/d96d9aad52e010da977d813f479494397ee53ec6/packages/%40ember/-internals/glimmer/lib/helpers/action.ts#L86-L113 and https://github.com/emberjs/ember.js/blob/d96d9aad52e010da977d813f479494397ee53ec6/packages/%40ember/-internals/glimmer/lib/helpers/mut.ts#L81

Reply:
Your second example uses the `fn` helper so can we try using `fn` instead?

Reply:
why? :)

Reply:
```suggestion
      {{on "input" this.setHighValue}}>
```

Then in the `setHighValue` and `setLowValue` actions the first argument should be the input event. You can get `target.value` from there.

Reply:
https://emberjs.github.io/rfcs/0470-fn-helper.html

Ember is moving away from overusage of the action helper